app/src/main/java/org/parasol/resources/ClaimWebsocketChatBot.java

```diff
@@ -43,6 +44,7 @@ public ClaimBotQueryResponse onError(Throwable error) {
 
     @OnTextMessage
     @WithSpan("ChatMessage")
+    @Blocking
```

Comment:
Does doing this, though, make the chat synchronous, meaning the response no longer streams? If so, then we don't want to do this as its an unintended side-effect.